dev/ts/pages/diagram-navigation.ts

```diff
@@ -26,37 +26,37 @@ function configureFitAndZoomButtons(): void {
 }
 
 function configureZoomThrottleInput(parameters: URLSearchParams): HTMLInputElement {
-  const elZoomThrottle = document.getElementById('zoom-throttle') as HTMLInputElement;
+  const elementZoomThrottle = document.getElementById('zoom-throttle') as HTMLInputElement;
```

Comment:
**suggestion:** rename into `zoomThrottleElement` for consistency with the type and the common usage.